client/state/sharing/publicize/reducer.js

```diff
@@ -57,7 +65,7 @@ export function connections( state = {}, action ) {
 		case SERIALIZE:
 			return state;
 		case DESERIALIZE:
-			return state;
+			return isValidStateWithSchema( state, connectionsSchema ) ? state : {};
```

Comment:
I'm assuming line length is what drove you to using ternary in this case and condition blocks in the others, though from a legibility perspective, I think we should be consistent (on the block?).